data_scripts/get_camera_points.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python
+import os
+import sys
+import csv
+from datetime import datetime
+
+from django.core.wsgi import get_wsgi_application
+# from django.contrib.gis.geos import GEOSGeometry
+from django.contrib.gis.geos import Point
+
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "streamwebs_frontend.settings")
+# Set proj path to be relative to data_scripts directory
+proj_path = "../streamwebs_frontend/"
+sys.path.append(proj_path)
+application = get_wsgi_application()
+
+from streamwebs.models import Site  # NOQA
+from streamwebs.models import CameraPoint # NOQA
+
+
+# Function to convert the (string) coordinates to a float value
+def num(x):
+    try:
+        return float(x)
+    except ValueError:
+        return x
+
+
+if os.path.isdir("../sw_data/"):
+    datafile = '../sw_data/camera_points.csv'
+else:
+    datafile = '../csvs/camera_points.csv'
+
+
+# Title, Post date, Updated date, Marker, Coordinates, Lat, Long
+with open(datafile, 'r') as csvfile:
+    sitereader = csv.reader(csvfile)
+    for row in sitereader:
```

Comment:
I probably should update the names here to reflect what we're importing. I copied the get_sites.py as a template.